crate/guides/quickstart.md

```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
 
 ## Setup
 
-This framework requires you to install [Rust](https://www.rust-lang.org/tools/install).
+The two fundamental things to setup:
 
-You'll need a recent version of Rust: `rustup update`
+ * [Rust](https://www.rust-lang.org/tools/install), the language itself.
+  * `rustup update` is also important
```

Comment:
There is probably an extra white-space before `*`.

And are you sure that users will know that they should run it in the  terminal? => I would change _"is also important"_ to something more valuable to users. (And _"is also important"_ statement isn't true if the user just installed Rust).